packages/web-app-files/src/helpers/share/index.ts

```diff
@@ -1,6 +1,2 @@
 export * from './link'
-export * from './permission'
-export * from './role'
-export * from './share'
-export * from './status'
-export * from './type'
+export * from 'web-client/src/helpers/share/type'
```

Comment:
is this intended? 🤔  I thought you now import everything share related from the new location.

Reply:
Nope, automatic refactor failure -_-'